set.mm

```diff
@@ -707210,8 +707244,8 @@ Ordering on reals (cont.) - extension
 -.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
 $)
 
-  $( Addition and subtraction on one side of 'less or equal'.  (Contributed by
-     Alexander van der Vekens, 18-Mar-2018.) $)
+  $( Addition and subtraction on one side of "less-than-or-equal-to".
```

Comment:
Either "less than or equal to" or less-than-or-equal-to.

Reply:
Done.